src/main/java/cn/nukkit/block/BlockNetherReactor.java

```diff
@@ -0,0 +1,45 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.item.ItemDiamond;
+import cn.nukkit.item.ItemIngotIron;
+import cn.nukkit.item.ItemTool;
+
+/**
+ * Created by Leonidius20 on 18.08.18.
+ */
+public class BlockNetherReactor extends BlockSolid {
```

Comment:
this is removed from mc

Reply:
My bad. I will delete it.

Reply:
Actually, this block still works! ![Nether Reactor Core](https://preview.ibb.co/fdwGse/Screenshot_2018_08_21_16_30_38.png)

Reply:
Yes, the block still exists. I don't mind if you add it

Reply:
But I'm not sure does this block still have features like dropping 3 iron ignots and 3 diamonds, or is an iron pixage still required to break it. So I better delete this class.